add-on/chrome/bootstrap.js

```diff
@@ -75,6 +75,21 @@ var WindowListener = {
       },
 
       /**
+       * @return {Promise} Getter for the Loop constants
+       */
+      get constants() {
+        if (!this._constants) {
+          this.LoopAPI.sendMessageToHandler({
```

Comment:
Can you add a comment here that this request is OK/ synchronous, even though it's using a callback? Otherwise ppl might get confused...